src/composables/search.js

```diff
@@ -28,6 +33,18 @@ export const useSearchStore = defineStore('search', () => {
     return await $fetch(`/api/search/${id}`)
   }
 
+  async function isNameAvailable(name) {
```

Comment:
Confusing naming, I understand "available" as can be claimed, but this name is already claimed. I would name it `isNameClaimed`, `isNameExist` or `isNameRegistered`